origin_ca.go

```diff
@@ -0,0 +1,147 @@
+package cloudflare
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// OriginCA resource
+// This is the origin_ca resource definition (as defined here:
+// https://api.cloudflare.com/#origin-ca || https://api.cloudflare.com/#cloudflare-ca
```

Comment:
It seems like the `#origin-ca` anchor no longer works, so we might as well remove it.